docs/components/modeler/web-modeler/process-applications.md

```diff
@@ -52,7 +52,6 @@ You should be aware of the following limitations when working with process appli
 ### General limitations
 
 - You cannot create folders inside a process application folder.
-- Once created, you cannot modify, delete, or roll back a process application version.
 - Self-Managed does not support defining cluster stages, identifying clusters by tags, or cluster promotion.
```

Comment:
@nithinssabu https://github.com/camunda/product-hub/issues/2073 addressed this, right?

Reply:
Delete and rollback (I assume this is restore) is possible. Modifying a version is not possible, but I don't see it as a limitation.